distribution/pom.xml

```diff
@@ -1,368 +1,373 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
this file shows too many changes - could you try to make the PR to only show the newly added lines? Thanks!